src/tools/kdb/complete.cpp

```diff
@@ -24,12 +24,11 @@ CompleteCommand::CompleteCommand ()
 {
 }
 
+/*
+ * McCabe complexity of 11, 1 caused by verbose switch so actually its ok
+ */
 int CompleteCommand::execute (const Cmdline & cl)
 {
-	if (cl.arguments.size () != 1)
```

Comment:
without arguments is like calling kdb complete "" (basically that means show me every possible completion if i haven't typed anything yet, which will be every possibility so namespaces, keys (expanded to nodes) and mountpoints). I'll document it